admin-client/src/components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import './LoginForm.css';
+
+const LoginForm = () => (
+    <div>
+        <p className='admin-text'>Atom Space Admin Panel</p>
+        <form>
+            <input className='login' type="text" placeholder='Login' />
+            <input className='password' type="password" placeholder='Password' />
+            <input className='submit-input' type="submit" onclick="" value="Log in" />
+        </form>
+    </div>
+);
```

Comment:
connect this component in `redux-form`
https://redux-form.com/8.2.2/docs/gettingstarted.md/
```
ContactForm = reduxForm({
  // a unique name for the form
  form: 'contact'
})(ContactForm)
```